pkg/nuctl/command/invoke.go

```diff
@@ -55,6 +56,18 @@ func newInvokeCommandeer(rootCommandeer *RootCommandeer) *invokeCommandeer {
 				return errors.New("Invalid logger level name. Must be one of none / debug / info / warn / error")
 			}
 
+			// convert via
+			switch commandeer.invokeVia {
+			case "any":
+				commandeer.invokeOptions.Via = platform.InvokeViaAny
+			case "eip":
```

Comment:
Is `eip` a known name? If not maybe change to something more readable?

Reply:
Changed to `external-ip`